examples/example.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
I am sorry but this is not a valid JSON. Strings need quotes, also a coma "," is needed between elements, for arrays you have to put "[ ]" around all the elements, something like this

```
[{
	"origin": "Barcelona",
	"destination": "Pekin",
	"transport": "airport bus",
	"gate": "J",
	"seat": "A12"
}, {
	"origin": "Barcelona",
	"destination": "Pekin",
	"transport": "airport bus",
	"gate": "H",
	"seat": "0"
}]

```
Here you can check if a JSON is valid (among other sites) 
[JSONLINT](https://jsonlint.com/)

You can learn more about JSON files in a lot of sites, this is not very difficult, i think
https://www.w3schools.com/js/js_json_intro.asp